dev/content/pages/About.md

```diff
@@ -105,15 +105,18 @@ save_as: index.html
 <!-- Left -->
 <div class="about-box">
 <h1 style="margin-top:0; text-align: left;">About Me</h1>
-    <p>I am a data scientist at Google where I use data to inform decisions, although all my opinions here are my own. I have previously worked at Sweetgreen, designing systems that would benefit team members and communities through sustainable and healthy food, and SpaceX, creating tools that would ultimately launch rocket ships. I feel fortunate to have worked with amazing teams throughout my career.</p>
-    <p>There are 3 things I strive to be:</p>
+    <p>I am a data scientist at Google. My focus is supporting strategic decision making with data, although all my opinions here are my own. Previously worked at Sweetgreen and SpaceX, doing the same, using data to inform staffing strategy and rocket launch cadence.</p>
```

Comment:
I mainly wanted to corrected "Previously, I" to make it an actual grammatical sentence but threw in other suggestions.
```suggestion
    <p>I am a data scientist at Google. My focus is supporting strategic decision making with data, although all my opinions here are my own. Previously, I worked at Sweetgreen and SpaceX, doing the same line of work as now but on different applications, which is using data to inform staffing strategy and rocket launch cadence.</p>
```